lib/oban/query.ex

```diff
@@ -84,6 +84,27 @@ defmodule Oban.Query do
     :ok
   end
 
+  @spec retry_all_jobs(Config.t()) :: :ok
+  def retry_all_jobs(conf) do
+    query =
+      Job
+      |> where([j], j.state not in ~w<available executing scheduled completed>)
```

Comment:
I think you don't want to retry `cancelled` jobs, also.

Maybe using `in` instead of `not in` would be a better fit here too